code/script.js

```diff
@@ -28,7 +28,7 @@ do {
     alert('Your choice is not available, Please try again');
   }
 }
-while (foodChoiceNum >= 4); //do...while loop if user does not enter a valid option (1,2 or 3) they are asked again. this only checks for numbers, if they type a word it doesnt work and goes to pizza. I don't know how to fix this
+while (foodChoiceNum >= 4); //do...while loop if user does not enter a valid option (1,2 or 3) they are asked again. this only checks for numbers, if they type a word it doesnt work and goes to pizza. I don't know how to fix this, though Diego has given me some tips!
```

Comment:
I really like the way you use the "do .. while" loop for invalid numbers. It feels intuitive to try again when I press a wrong number when ordering. I would love to know if you find a solution for typing in letters.